### PR TITLE
Set up local development via Docker (issue #135)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: distributionviewer/views.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        # Check the database by counting Metrics.
+        Metric.objects.count()
+        # TODO: Add checks for other dependencies as we add them
+        return HttpResponse('ok')
+    except:
+        return HttpResponseServerError('database failure')
+
+
+def ops_version(request):
+    # TODO: Generate this via TravisCI.
```
Comment (openjck) [inline distributionviewer/views.py:25]: Is this now CircleCI?

Comment (robhudson) [reply distributionviewer/views.py:25]: Yep, I'll update the comment. Thanks.

Comment (robhudson) [reply distributionviewer/views.py:25]: I'll fix this in the next iteration when I actually do the TODO. It's a minor comment.

File: docs/docker.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+already. See the [Docker documentation][docker-docs] for help.
+
+[docker-docs]: https://docs.docker.com/
+
+* To launch the containers:
+
+  `docker-compose up`
+
+  Running `docker-compose up` from the root directory starts the database and
+  web server. The frontend build step is the same as documented in the README
```
Comment (openjck) [inline docs/docker.md]: We could link to _frontend.md_ here.

File: distributionviewer/views.py (hunk 1 above)
Comment (openjck) [reply distributionviewer/views.py:25]: Totally, just wanted to be sure I understood correctly.

**Pull request merged**